source/clear-linux/guides/maintenance/swupd-guide.rst

```diff
@@ -0,0 +1,160 @@
+.. _swupd-guide:
+
+Use swupd
+#########
+
+*swupd* manages the software update capability of |CLOSIA|. It can check for
+valid system updates and, if found, download and install them. It can also
+perform verification of the system software. 
+
+|CL| uses :ref:`bundles-about<bundles>` as the base abstraction for
+installing functionality on top of the core operating system. Use the `swupd`
+tool to install and remove bundles.
+
+.. contents:: 
+   :local:
+   :depth: 2
+
+.. _man_swupd:
+
+For a full listing of commands and options please see the man page in |CL|
+
+.. code-block:: bash
+
+   man swupd
+
+OS Update and Verification
+**************************
+
+|CL| is designed to promote a regular update cadence. `swupd` helps to
+make sure that process is simple and secure.
+
+OS Info
+=======
+
+Current OS version and update server info:
+
+.. code-block:: bash
+
+   swupd info
+
+.. code-block:: console
+   Installed version: 23330
+   Version URL:       https://download.clearlinux.org/update/
+   Content URL:       https://cdn.download.clearlinux.org/update/
+
+Disable Automatic Updates
```

Comment:
To add to my point above, this subsection could be called "Enable or disable automatic updates."
(BTW: Most of the CL documentation uses title case, so it would be great if you made that consistent here.  It's a completely minor issue.)